go.mod

```diff
@@ -1,29 +1,26 @@
 module github.com/docker/compose/v2
 
-go 1.16
+go 1.17
```

Comment:
This may also print a warning that Go 1.17 is the minimum version supported; probably ok (not sure if anything is using the modules from this repository as a dependency), but it's common to support at least "latest" + "latest-1"

Reply:
Maybe we should put a warning in the README, tell people not to depend on this cli, or just move everything to `internal`

Reply:
Perhaps just a warning would be ok. `internal` always feels a bit like a "big hammer" to me (we _forbid_ you to use this code!)

That said; trying to only export things that are relatively / considered stable is good practice (also helps the mental overhead to know ("this function/const/variable is not exported, so I can change this without side-effects". My IDE also is better at pointing me at unused functions if they're not exported)